client/src/components/App/Content/LandingPage/adminLandingPage/desksFilterCard/desksFilterCard.tsx

```diff
@@ -17,10 +19,14 @@ const DesksFilterCard = (props: Props): JSX.Element => {
 
     const classes = useStyles();
     const { onUpdateButtonClicked } = props;
-    const { desks, isLoading, filteredDesks, clearAllDesks, onDeskClicked } = useDesksFilterCard();
+    const { filteredDesks, clearAllDesks, onDeskClicked } = useDesksFilterCard();
+
+    const displayedCounty = useSelector<StoreStateType, number>(state => state.user.displayedCounty);
+    const desks = useSelector<StoreStateType, Desk[]>(state => state.desk);
+    const countyDesks = useMemo(() => desks.filter(desk => desk.county === displayedCounty), [desks, displayedCounty]);
```

Comment:
Same for here, if this is used so many times shouldn't it be moved to a util or something?